src/Streamly/Internal/Data/Array/Foreign/Mut/Type.hs

```diff
@@ -677,10 +679,21 @@ roundUpLargeArray size =
             ((size + blockSize - 1) .&. negate blockSize)
     else size
 
-{-
 roundUpToPower2 :: Int -> Int
-roundUpToPower2 = undefined
--}
+roundUpToPower2 x =
+#if WORD_SIZE_IN_BITS == 64
+    1 + _z6
```

Comment:
Why is it named `_z6`? We are testing on 64-bit platforms so it should not be unused.